tracecat/auth/users.py

```diff
@@ -193,11 +233,16 @@ async def list_users(*, session: SQLModelAsyncSession) -> list[User]:
 
 
 def default_admin_user() -> UserCreate:
+    if not (email := os.getenv("TRACECAT__SETUP_ADMIN_EMAIL")):
+        raise ValueError("TRACECAT__SETUP_ADMIN_EMAIL is not set")
+    if not (password := os.getenv("TRACECAT__SETUP_ADMIN_PASSWORD")):
+        raise ValueError("TRACECAT__SETUP_ADMIN_PASSWORD is not set")
+
```

Comment:
i know i wrote this, but does this break the app on restart if you set the initial admin user at the start, then throw away these env vars?

Reply:
Yeah good point it does. How can I check if an admin user already exists?

Reply:
This should only run when there are no users:
```tracecat/api/setup_defaults.py
async def setup_defaults(session: AsyncSession, admin_role: Role):
    users = await list_users(session=session)
    if len(users) == 0:
        # Create admin user only if there are no users
        await get_or_create_default_admin_user()
```